prism/src/main/java/org/apache/falcon/jdbc/MonitoringJdbcStateStore.java

```diff
@@ -67,82 +70,88 @@ public MonitoredFeedsBean getMonitoredFeed(String feedName){
         } finally {
             entityManager.close();
         }
-        return ((MonitoredFeedsBean)result.get(0));
+        return ((MonitoredEntityBean)result.get(0));
     }
 
-    public void deleteMonitoringFeed(String feedName) {
+    public void deleteMonitoringFeed(String entityName, String entityType) {
```

Comment:
To pass the entity type to the query you just need to do entityType.name() and to convert to EntityType you need to do EntityType.valueOf(). This will ensure that invalid values are not passed.